### ISSUE TITLE
Memory object 

### ISSUE THREAD

**Issue by Faliszek** [cawfree/react-native-webassembly#9]
First of all - thank you so much for your work on this, amazing job! 

I try to use it in order to load some wasm file generated from this repo: https://github.com/matrix-org/matrix-rust-sdk/tree/main/bindings/matrix-sdk-crypto-js

It loads perfectly fine, but later this library uses `memory` object which from what I read, in browser returns from instatiate function (I think)

When I compile their wasm file to wat there is a code that looks like this

```wat
(memory $memory (export "memory") 20)
```

which according to chatgpt should be accesible on js side like this

```js
//loading binary
const result = await WebAssembly.instatiate(binary);
//getting memory.buffer
result.exports.memory.buffer
```

Is there a plan for support this behaviour? Or do you have any guidance how can I add it? Thanks in advance :)

**Comment by cawfree** [comment]
Hey @Faliszek, thanks!

You're correct, this functionality isn't implemented yet, but it would be very nice to have. Thanks for sharing!

I'll need to have a little think about how to implement something like this, but it should be possible.

**Comment by Faliszek** [comment]
Let me know if it's there anything I can help with. Once again thanks for all your work, this library is a game changer for me.

**Comment by cawfree** [comment]
This functionality has been implemented as published as version [`0.3.3`](https://www.npmjs.com/package/react-native-webassembly). ([Instructions](https://github.com/cawfree/react-native-webassembly#-memory))

Thank you for your patience whilst this was being implemented.

If you have any trouble with this feature or notice any other aspects of the specification that are missing, please feel free to raise another issue!

https://github.com/cawfree/react-native-webassembly/pull/10
